service_discord_youtube.bat

```diff
@@ -13,3 +45,6 @@ sc delete "%SRVCNAME%"
 sc create "%SRVCNAME%" binPath= "%~dp0winws.exe %ARGS%" DisplayName= "zapret DPI bypass : winws1" start= auto
 sc description "%SRVCNAME%" "zapret DPI bypass software"
 sc start "%SRVCNAME%"
+
+echo Серис был установлен и запущен.

```

Comment:
You cannot claim that the service was successfully launched. Just leave a pause without a message.